crates/sui-adapter/src/execution_engine.rs

```diff
@@ -260,7 +260,7 @@ fn execution_loop<
                 // Charge gas for this VM execution
                 gas_status.charge_vm_gas()?;
 
-                let module_id = ModuleId::new(package.0.into(), module);
+                let module_id = ModuleId::new(package.into(), module);
```

Comment:
This is why we don't need to do much special -- the package gets wrapped up in a `ModuleId` so it loses its version and digest almost immediately.